model/project_ref.go

```diff
@@ -694,6 +713,21 @@ func FindMergedProjectRefsByRepoAndBranch(owner, repoName, branch string) ([]Pro
 	return addLoggerAndRepoSettingsToProjects(projectRefs)
 }
 
+// FindAllMergedProjectRefByRepoAndBranch finds a hidden ProjectRef with matching repo/branch, and merges repo information.
+func FindAllMergedProjectRefByRepoAndBranch(owner, repoName, branch string) ([]ProjectRef, error) {
+	projectRefs := []ProjectRef{}
+
+	q := byOwnerRepoAndBranch(owner, repoName, branch)
+	q[projectRefUseRepoSettingsKey] = true
```

Comment:
what's the reason projects that aren't using repo settings should be excluded from this?

Reply:
This is poorly named; at this point I only want project refs that use repo settings. I'll rename.